# Russian translations for plugins package.
# Copyright (C) 2011 THE plugins'S COPYRIGHT HOLDER
# This file is distributed under the same license as the plugins package.
# Automatically generated, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: enigma2-vision-core\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-03-18 15:16+0330\n"
"PO-Revision-Date: 2013-10-20 17:17+0100\n"
"Last-Translator: Andy Blackburn <andyblac@o2.co.uk>\n"
"Language-Team: RU locale by Sowa\n"
"Language: ru_RU\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: Poedit 1.5.5\n"
"X-Poedit-SourceCharset: UTF-8\n"

#: ../src/BackupManager.py:171 ../src/BackupManager.py:305
#: ../src/ImageManager.py:157 ../src/ImageManager.py:313
msgid "%a %e %b  %-H:%M"
msgstr "%a %e %b  %-H:%M"

#: ../src/ImageManager.py:287
msgid "'User' url has not been specified, please use MENU button to initialise"
msgstr ""

#: ../src/SwapManager.py:311
msgid "128 Mb"
msgstr ""

#: ../src/SwapManager.py:311
msgid "16 Mb"
msgstr "16 МБайт"

#: ../src/SwapManager.py:311
msgid "256 Mb"
msgstr ""

#: ../src/BackupManager.py:49 ../src/ImageManager.py:52
msgid "30 Days"
msgstr "30 Дней"

#: ../src/SwapManager.py:311
msgid "32 Mb"
msgstr "32 МБайта"

#: ../src/SwapManager.py:311
msgid "64 Mb"
msgstr "64 МБайта"

#: ../src/SwapManager.py:311
msgid "8 Mb"
msgstr "8 МБайт"

#: ../src/SwapManager.py:311
msgid "96 Mb"
msgstr "96 МБайт"

#: ../src/BackupManager.py:517 ../src/RestoreWizard.py:159
#: ../src/RestoreWizard.py:270
#, fuzzy
msgid "A background update check is in progress, please try again."
msgstr "Выполняется фоновая проверка обновлений, пожалуйста, ждите"

#: ../src/SwapManager.py:129 ../src/SwapManager.py:252
msgid "Activate"
msgstr "Активировать"

#: ../src/SwapManager.py:127 ../src/SwapManager.py:248
msgid "Active"
msgstr "Активный"

#: ../src/SoftcamManager.py:96 ../src/SoftcamManager.py:169
msgid "Active:"
msgstr "Активный:"

#: ../src/IPKInstaller.py:135
msgid "Are you ready to install ?"
msgstr "Вы готовы к инсталляции?"

#: ../src/Multibootmgr.py:140
#, fuzzy, python-format
msgid "Are you sure you want to delete image slot %s ?"
msgstr ""
"Вы действительно хотите удалить этоу копию:\n"
" "

#: ../src/BackupManager.py:313
msgid ""
"Are you sure you want to delete this backup:\n"
" "
msgstr ""
"Вы действительно хотите удалить этоу копию:\n"
" "

#: ../src/ImageManager.py:321
#, fuzzy
msgid ""
"Are you sure you want to delete this image backup:\n"
" "
msgstr ""
"Вы действительно хотите удалить этоу копию:\n"
" "

#: ../src/ImageManager.py:1513
msgid ""
"Are you sure you want to download this image:\n"
" "
msgstr "Вы действительно хотите загрузить имидж:"

#: ../src/ImageManager.py:396
#, fuzzy
msgid ""
"Are you sure you want to flash this image:\n"
" "
msgstr "Вы действительно хотите загрузить имидж:"

#: ../src/BackupManager.py:376
msgid ""
"Are you sure you want to restore this backup:\n"
" "
msgstr ""
"Вы действительно хотите восстановить этоу копию:\n"
" "

#: ../src/SoftcamManager.py:121 ../src/SwapManager.py:131
msgid "Autostart"
msgstr "Автостарт"

#: ../src/ImageManager.py:797
msgid "Backing up eMMC partitions for USB flash ..."
msgstr ""

#: ../src/ImageManager.py:801
msgid "Backing up eMMC partitions for USB flash..."
msgstr ""

#: ../src/BackupManager.py:1165 ../src/BackupManager.py:1169
#: ../src/BackupManager.py:1181 ../src/BackupManager.py:1185
msgid "Backing up files..."
msgstr "Резервное копирование файлов..."

#: ../src/ImageManager.py:781 ../src/ImageManager.py:785
#, fuzzy
msgid "Backing up kernel..."
msgstr "Резервное копирование файлов..."

#: ../src/ImageManager.py:789 ../src/ImageManager.py:793
#, fuzzy
msgid "Backing up root file system..."
msgstr "Резервное копирование файлов..."

#: ../src/ImageManager.py:829
#, fuzzy
msgid "Backup complete."
msgstr "Резервное копирование завершено..."

#: ../src/BackupManager.py:1189
#, fuzzy
msgid "Backup complete..."
msgstr "Резервное копирование завершено..."

#: ../src/ImageManager.py:352
#, fuzzy
msgid "Backup confirmation"
msgstr "Подтверждение создания резервной копии"

#: ../src/BackupManager.py:984
#, fuzzy
msgid "Backup created"
msgstr "Создание резервной копии"

#: ../src/BackupManager.py:1358
#, fuzzy
msgid "Backup failed - e. g. wrong backup destination or no space left on backup device."
msgstr "Резервное копирование неудачно - возможно, неправильное расположение или нет свободного места на выбранном устройстве"

#: ../src/BackupManager.py:360
#, fuzzy
msgid ""
"Backup in progress,\n"
"Please wait for it to finish, before trying again."
msgstr ""
"Копия создаётся...\n"
"Дождитесь окончания перед следующим действием"

#: ../src/BackupManager.py:131 ../src/BackupManager.py:197
#: ../src/BackupManager.py:329 ../src/BackupManager.py:344
#: ../src/BackupManager.py:387 ../src/BackupManager.py:405
#: ../src/BackupManager.py:1147 ../src/ui.py:67 ../src/ImageManager.py:372
#, fuzzy
msgid "Backup manager"
msgstr "Резервироание настроек"

#: ../src/SoftcamManager.py:238
msgid "CCcam can't run whilst MGcamd is running"
msgstr "CCcam не работает одновременно с MgCamd"

#: ../src/SoftcamManager.py:295
#, fuzzy
msgid "CCcam can't run whilst MGcamd is running."
msgstr "CCcam не работает одновременно с MgCamd"

#: ../src/MountManager.py:287
#, fuzzy
msgid "Can't un-mount the partition; make sure it is not being used for SWAP or record/timeshift paths."
msgstr "Не можете размонтировать раздел, убедитесь, что он не используется для подкачки или в путях записи/TimeShift"

#: ../src/BackupManager.py:744 ../src/BackupManager.py:844
#: ../src/BackupManager.py:943 ../src/Multibootmgr.py:62
#: ../src/Multibootmgr.py:84 ../src/MountManager.py:353
msgid "Cancel"
msgstr "Отмена"

#: ../src/IPKInstaller.py:99 ../src/IPKInstaller.py:103
#, fuzzy
msgid "Change location"
msgstr "Изменить местоположение"

#: ../src/ImageManager.py:773
#, fuzzy
msgid "Checking free RAM.."
msgstr "Проверка свободной памяти.."

#: ../src/SoftcamManager.py:635
msgid "Checking softcams..."
msgstr "Проверка эмуляторов..."

#: ../src/BackupManager.py:945
#, fuzzy
msgid "Choose files"
msgstr "Выбор файлов"

#: ../src/BackupManager.py:946
#, fuzzy
msgid "Choose local IPK's folder"
msgstr "Выберите локальную папку содержащую пакеты ipk"

#: ../src/MountManager.py:339
msgid "Choose where to mount your devices to:"
msgstr "Выбор точки монтирования:"

#: ../src/ui.py:78 ../src/ImageManager.py:1361 ../src/SoftcamManager.py:118
#: ../src/IPKInstaller.py:65 ../src/IPKInstaller.py:187
#: ../src/SwapManager.py:128
msgid "Close"
msgstr "Закрыть"

#: ../src/BackupManager.py:427 ../src/BackupManager.py:431
msgid "Comparing against backup..."
msgstr "Сравнение с резервной копией..."

#: ../src/ImageManager.py:1481
msgid "Compress"
msgstr ""

#: ../src/SwapManager.py:130 ../src/SwapManager.py:192
msgid "Create"
msgstr "Создать"

#: ../src/ui.py:72
#, fuzzy
msgid "Create and Manage your SWAP files."
msgstr "Создание и управление файлом подкачки"

#: ../src/ui.py:68
#, fuzzy
msgid "Create and flash complete images of your system."
msgstr "Создание и восстанавление полных образов системы"

#: ../src/ui.py:74
msgid "Create empty slot."
msgstr ""

#: ../src/ImageManager.py:975 ../src/ImageManager.py:978
#: ../src/ImageManager.py:981 ../src/ImageManager.py:983
#: ../src/ImageManager.py:985
#, fuzzy
msgid "Create:"
msgstr "Создать"

#: ../src/ImageManager.py:777
#, fuzzy
msgid "Creating SWAP.."
msgstr "Создание файла подкачки..."

#: ../src/BackupManager.py:419 ../src/BackupManager.py:423
#: ../src/BackupManager.py:1157 ../src/BackupManager.py:1161
msgid "Creating list of installed plugins..."
msgstr "Создать список установленных плагинов..."

#: ../src/ImageManager.py:821 ../src/ImageManager.py:825
msgid "Creating zip..."
msgstr ""

#: ../src/SwapManager.py:154 ../src/SwapManager.py:248
#: ../src/SwapManager.py:253
#, fuzzy
msgid "Current status:"
msgstr "Текущее состояние:"

#: ../src/BackupManager.py:49 ../src/ImageManager.py:52
msgid "Daily"
msgstr "Ежедневно"

#: ../src/SwapManager.py:247
msgid "Deactivate"
msgstr "Деактивировать"

#: ../src/BackupManager.py:156 ../src/ImageManager.py:137
#: ../src/SwapManager.py:202
msgid "Delete"
msgstr "Удалить"

#: ../src/BackupManager.py:778
msgid "Deselect"
msgstr "Отмена выбора"

#: ../src/BackupManager.py:248 ../src/BackupManager.py:268
#: ../src/ImageManager.py:262 ../src/ImageManager.py:271
#: ../src/MountManager.py:130 ../src/MountManager.py:265
#: ../src/MountManager.py:280
msgid "Device: "
msgstr "Место хранения: "

#: ../src/ImageManager.py:238
msgid "Device: None available"
msgstr "Место хранения: Не выбрано!"

#: ../src/BackupManager.py:229
#, fuzzy
msgid "Device: none available"
msgstr "Место хранения: Не выбрано!"

#: ../src/SoftcamManager.py:165
#, fuzzy
msgid "Disable startup"
msgstr "Отключить автостарт"

#: ../src/ImageManager.py:279
msgid "Do you want to change download url"
msgstr ""

#: ../src/ImageManager.py:350
msgid ""
"Do you want to create a full image backup?\n"
"This can take about 6 minutes to complete."
msgstr ""

#: ../src/ImageManager.py:392
#, fuzzy, python-format
msgid ""
"Do you want to flash image\n"
"%s"
msgstr "Вы действительно хотите загрузить имидж:"

#: ../src/IPKInstaller.py:148
#, fuzzy
msgid "Do you want to restart GUI now ?"
msgstr "Вы хотите восстановить сохранённые плагины?"

#: ../src/BackupManager.py:664
msgid "Do you want to restore your Enigma2 plugins ?"
msgstr "Вы хотите восстановить сохранённые плагины?"

#: ../src/BackupManager.py:459
#, fuzzy
msgid "Do you want to restore your enigma2 settings ?"
msgstr "Вы хотите восстановить сохранённые настройки?"

#: ../src/ImageManager.py:1483
#, fuzzy
msgid "DownLoad"
msgstr "Загрузить"

#: ../src/ImageManager.py:1362
msgid "Download"
msgstr "Загрузить"

#: ../src/ImageManager.py:1515
#, fuzzy
msgid "Download confirmation"
msgstr "Подтверждение загрузки"

#: ../src/ImageManager.py:1532
#, fuzzy
msgid "Downloading"
msgstr "Загрузить"

#: ../src/ImageManager.py:136 ../src/ImageManager.py:1345
msgid "Downloads"
msgstr "Скачать"

#: ../src/ImageManager.py:468
msgid "ET8500 Image Restore"
msgstr ""

#: ../src/ImageManager.py:466
msgid ""
"ET8500 Multiboot: Yes to restore OS1 No to restore OS2:\n"
" "
msgstr ""

#: ../src/Multibootmgr.py:127
msgid "Empty slot"
msgstr ""

#: ../src/SwapManager.py:255
#, fuzzy
msgid "Enable SWAP at startup"
msgstr "Включить автостарт файла подкачки"

#: ../src/SoftcamManager.py:167
#, fuzzy
msgid "Enable startup"
msgstr "Включить автостарт"

#: ../src/BackupManager.py:1111 ../src/ImageManager.py:651
#, fuzzy
msgid "Enough retries, delaying till next schedule."
msgstr "Достаточно повторов, задержка до следующего пункта планировщика."

#: ../src/Multibootmgr.py:87
msgid "Erase"
msgstr ""

#: ../src/RestoreWizard.py:93
msgid "Exit the restore wizard"
msgstr "Выход из Мастера восстановления"

#: ../src/ImageManager.py:1481
msgid "Expand"
msgstr ""

#: ../src/IPKInstaller.py:94 ../src/IPKInstaller.py:113
msgid "Extra IPK's"
msgstr "Дополнительные пакеты ipk"

#: ../src/ImageManager.py:134
msgid "Flash"
msgstr ""

#: ../src/ImageManager.py:398
#, fuzzy
msgid "Flash confirmation"
msgstr "Подтверждение загрузки"

#: ../src/ImageManager.py:464
msgid "Flash image unzip successful."
msgstr ""

#: ../src/SoftcamManager.py:260 ../src/SoftcamManager.py:314
#, fuzzy
msgid "Found non-standard softcam, trying to start, this may fail."
msgstr "Найден нестандартный эмулятор, возможны проблемы при запуске"

#: ../src/ImageManager.py:262
msgid "Free space:"
msgstr "Свободно:"

#: ../src/MountManager.py:117
msgid "GB"
msgstr "ГБайт"

#: ../src/H9SDmanager.py:34
#, fuzzy
msgid "H9 SDcard manager"
msgstr "Управление эмуляторами"

#: ../src/ui.py:76
#, fuzzy
msgid "H9SDcard Manager"
msgstr "Файл подкачки"

#: ../src/MountManager.py:70
msgid "HARD DISK: "
msgstr "Жёсткий диск: "

#: ../src/IPKInstaller.py:37 ../src/IPKInstaller.py:180
#, fuzzy
msgid "IPK installer"
msgstr "Установщик IPK"

#: ../src/ImageManager.py:1411
#, python-format
msgid "Image Version %s"
msgstr ""

#: ../src/ui.py:68 ../src/ImageManager.py:111 ../src/ImageManager.py:183
#: ../src/ImageManager.py:341 ../src/ImageManager.py:362
#: ../src/ImageManager.py:767
#, fuzzy
msgid "Image manager"
msgstr "Управление прошивкой"

#: ../src/SwapManager.py:126 ../src/SwapManager.py:253
msgid "Inactive"
msgstr "Неактивный"

#: ../src/SwapManager.py:272 ../src/SwapManager.py:338
msgid "Info"
msgstr "Инфо"

#: ../src/Multibootmgr.py:150
msgid "Init SDCARD"
msgstr ""

#: ../src/Multibootmgr.py:65 ../src/Multibootmgr.py:89 ../src/H9SDmanager.py:58
msgid "Init SDcard"
msgstr ""

#: ../src/H9SDmanager.py:73
msgid "Init Zgemma H9 SDCARD - please reboot after use."
msgstr ""

#: ../src/H9SDmanager.py:97
msgid "Init Zgemma H9 USB/SDA1 - please reboot after use."
msgstr ""

#: ../src/IPKInstaller.py:66 ../src/IPKInstaller.py:188
msgid "Install"
msgstr "Установка"

#: ../src/ui.py:69
msgid "Install IPK's from your tmp folder."
msgstr "Установка IPK пакетов из папки /tmp"

#: ../src/IPKInstaller.py:137
#, fuzzy
msgid "Install confirmation"
msgstr "Подтверждение установки"

#: ../src/plugin.py:164
msgid "Install extensions."
msgstr "Установка расширений."

#: ../src/ui.py:69
msgid "Install local extension"
msgstr "Установка дополнений"

#: ../src/IPKInstaller.py:145
msgid "Installing..."
msgstr "Устанавливается..."

#: ../src/IPKInstaller.py:190
msgid "Invert"
msgstr "Изменить порядок"

#: ../src/plugin.py:182
msgid "Ipkg"
msgstr "Пакеты ipk"

#: ../src/IPKInstaller.py:97
#, fuzzy
msgid "It seems you have not setup an extra location. Please set it up in the Backup manager setup menu."
msgstr "Вы не указали путь для дополнительных пакетов ipk, укажите его в меню настроек резервного копирования"

#: ../src/BackupManager.py:969 ../src/SoftcamManager.py:545
msgid "Logs"
msgstr ""

#: ../src/ImageManager.py:262 ../src/MountManager.py:119
msgid "MB"
msgstr "МБайт"

#: ../src/SoftcamManager.py:254 ../src/SoftcamManager.py:310
#, fuzzy
msgid "MGcamd can't run whilst CCcam is running."
msgstr "MgCamd не работает одновременно с работающим CCcam"

#: ../src/ui.py:45 ../src/SoftcamManager.py:80
msgid "Main menu"
msgstr ""

#: ../src/ui.py:67
#, fuzzy
msgid "Manage the backups of your settings."
msgstr "Управление сохранениями настроек"

#: ../src/ui.py:70
#, fuzzy
msgid "Manage your devices mount points."
msgstr "Работа с точками монтирования"

#: ../src/MountManager.py:205
msgid "Mount"
msgstr "Монтировать"

#: ../src/MountManager.py:270
#, fuzzy
msgid "Mount failed."
msgstr "Монтирование неудачно!"

#: ../src/ui.py:70 ../src/MountManager.py:181
#, fuzzy
msgid "Mount manager"
msgstr "Менеджер монтирования"

#: ../src/MountManager.py:130 ../src/MountManager.py:264
#: ../src/MountManager.py:279
msgid "Mount: "
msgstr "Точка монтирования: "

#: ../src/ui.py:76
msgid "Move Nand root to SD card"
msgstr ""

#: ../src/ImageManager.py:813 ../src/ImageManager.py:817
#, fuzzy
msgid "Moving to backup Location..."
msgstr "Перейти к месту хранения..."

#: ../src/ui.py:74
#, fuzzy
msgid "MultiBoot manager"
msgstr "Менеджер монтирования"

#: ../src/ImageManager.py:534
msgid "Multiboot ERROR! - no STARTUP in boot partition."
msgstr ""

#: ../src/Multibootmgr.py:39
#, fuzzy
msgid "Multiboot image manager"
msgstr "Vision Диспетчер имиджей"

#: ../src/Multibootmgr.py:152
msgid "Multiboot manager - Cannot initialise SDcard when running image on SDcard."
msgstr ""

#: ../src/Multibootmgr.py:180
msgid "Multiboot manager - SDcard initialization run, please restart your Image."
msgstr ""

#: ../src/Multibootmgr.py:165
msgid "Multiboot manager - The SDcard must be at least 8MB."
msgstr ""

#: ../src/Multibootmgr.py:155
#, python-format
msgid "Multiboot manager - to use this routine %s image must be at OpenVision 4.2.043 or later and USB flashed - reply Yes to continue"
msgstr ""

#: ../src/ImageManager.py:472 ../src/ImageManager.py:1248
msgid "Multiboot only able to restore this backup to mmc slot1"
msgstr ""

#: ../src/RestoreWizard.py:104 ../src/RestoreWizard.py:107
msgid "NO, do not restore plugins"
msgstr "Нет, не восстанавливать плагины"

#: ../src/RestoreWizard.py:97
msgid "NO, do not restore settings"
msgstr "Нет, не восстанавливать настройки"

#: ../src/BackupManager.py:202 ../src/ImageManager.py:188
#, fuzzy
msgid "New backup"
msgstr "Новая копия"

#: ../src/BackupManager.py:171 ../src/BackupManager.py:173
#: ../src/BackupManager.py:305 ../src/BackupManager.py:307
#: ../src/ImageManager.py:157 ../src/ImageManager.py:159
#: ../src/ImageManager.py:313 ../src/ImageManager.py:315
#, fuzzy
msgid "Next backup: "
msgstr "Следующее резервирование: "

#: ../src/SoftcamManager.py:233 ../src/SoftcamManager.py:297
#, fuzzy
msgid ""
"No config files found, please setup CCcam first\n"
"in /etc/CCcam.cfg."
msgstr ""
"Отсутствуют файлы конфигурации, установите настройки CCcam\n"
"в /etc/CCcam.cfg"

#: ../src/SoftcamManager.py:241
#, fuzzy
msgid ""
"No config files found, please setup Hypercam first\n"
"in /etc/hypercam.cfg."
msgstr ""
"Отсутствуют файлы конфигурации, установите настройки Hypercam\n"
"в /etc/hypercam.cfg"

#: ../src/SoftcamManager.py:252 ../src/SoftcamManager.py:308
#, fuzzy
msgid ""
"No config files found, please setup MGcamd first\n"
"in /usr/keys."
msgstr ""
"Отсутствуют файлы конфигурации, установите настройки MGcamd\n"
"в /var/keys"

#: ../src/SoftcamManager.py:246
msgid ""
"No config files found, please setup Oscam first\n"
"in /etc/tuxbox/config"
msgstr ""
"Отсутствуют файлы конфигурации, установите настройки Oscam\n"
"в /etc/tuxbox/config"

#: ../src/SoftcamManager.py:303
#, fuzzy
msgid ""
"No config files found, please setup Oscam first\n"
"in /etc/tuxbox/config."
msgstr ""
"Отсутствуют файлы конфигурации, установите настройки Oscam\n"
"в /etc/tuxbox/config"

#: ../src/ImageManager.py:1369
msgid "No images found for selected download server...if password check validity"
msgstr ""

#: ../src/MountManager.py:95 ../src/MountManager.py:97
#: ../src/MountManager.py:108
msgid "None"
msgstr "Нет"

#: ../src/Multibootmgr.py:86
msgid "Note: slot list does not show current image or empty slots."
msgstr ""

#: ../src/BackupManager.py:681
msgid "Now skipping restore process"
msgstr "Пропустить процесс восстановления"

#: ../src/ImageManager.py:525
#, python-format
msgid ""
"OFGwrite error (also sent to any debug log):\n"
"%s"
msgstr ""

#: ../src/BackupManager.py:944 ../src/RestoreWizard.py:111
msgid "OK"
msgstr "ОК"

#: ../src/RestoreWizard.py:93
msgid "OK, to perform a restore"
msgstr "Нажмите ОК для восстановления"

#: ../src/BackupManager.py:900
msgid "Please enter a folder that contains some packages."
msgstr "Пожалуйста, введите путь к папке, которая содержит установочные пакеты."

#: ../src/SwapManager.py:304
#, fuzzy
msgid "Please select device to use as SWAP file location."
msgstr "Пожалуйста, выберите место расположения файла подкачки"

#: ../src/RestoreWizard.py:143
msgid "Please wait while plugins restore completes..."
msgstr "Ждите, идёт восстановление плагинов..."

#: ../src/MountManager.py:243
#, fuzzy
msgid "Please wait while scanning for devices..."
msgstr "Подождите, сканируются накопители..."

#: ../src/MountManager.py:362
#, fuzzy, python-format
msgid "Please wait while scanning your %s %s devices..."
msgstr "Подождите, сканируются накопители..."

#: ../src/RestoreWizard.py:198
msgid "Please wait while settings restore completes..."
msgstr "Ждите, идёт восстановление настроек..."

#: ../src/SoftcamManager.py:334
msgid "Please wait while starting\n"
msgstr "Ожидайте запуска\n"

#: ../src/SoftcamManager.py:451
msgid "Please wait while stopping\n"
msgstr "Ожидайте остановки\n"

#: ../src/ImageManager.py:450
#, python-format
msgid "Please wait while the flash prepares, after the image is flashed, your %s will restart - if error please use Recovery mode to restart."
msgstr ""

#: ../src/ImageManager.py:452
#, fuzzy
msgid "Please wait while the flash prepares."
msgstr "Пожалуйста подождите, идет сбор информации..."

#: ../src/RestoreWizard.py:131 ../src/RestoreWizard.py:136
#, fuzzy
msgid "Please wait while the system gathers information..."
msgstr "Пожалуйста подождите, идет сбор информации..."

#: ../src/MountManager.py:386
msgid "Please wait."
msgstr "Пожалуйста, ждите..."

#: ../src/BackupManager.py:1290
#, fuzzy
msgid "Plugin listing failed - e. g. wrong backup destination or no space left on backup device."
msgstr "Список плагинов не сохранён -возможно, неправильное расположение или нет свободного места на выбранном устройстве"

#: ../src/BackupManager.py:1173 ../src/BackupManager.py:1177
msgid "Preparing extra plugins..."
msgstr "Подготовка дополнительных плагинов..."

#: ../src/BackupManager.py:229 ../src/ImageManager.py:238
msgid "Press 'Menu' to select a storage device"
msgstr ""

#: ../src/Multibootmgr.py:63
msgid "Press Init to format SDcard."
msgstr ""

#: ../src/IPKInstaller.py:191
msgid "Press OK to toggle the selection."
msgstr "Нажмите OK для отметки/отключения выбора."

#: ../src/H9SDmanager.py:56
msgid "Press appropiate Init to move Nand root to SDcard."
msgstr ""

#: ../src/H9SDmanager.py:57
#, fuzzy
msgid "Reboot"
msgstr "Перезагрузка..."

#: ../src/BackupManager.py:451
msgid "Rebooting..."
msgstr "Перезагрузка..."

#: ../src/ImageManager.py:390
#, python-format
msgid ""
"Recording(s) are in progress or coming up in few seconds!\n"
"Do you still want to flash image\n"
"%s?"
msgstr ""

#: ../src/Multibootmgr.py:136
#, python-format
msgid "Removal of this slot will not show in %s Gui.  Are you sure you want to delete image slot %s ?"
msgstr ""

#: ../src/BackupManager.py:315 ../src/ImageManager.py:323
#: ../src/Multibootmgr.py:138 ../src/Multibootmgr.py:142
#: ../src/Multibootmgr.py:157
#, fuzzy
msgid "Remove confirmation"
msgstr "Подтверждение удаления"

#: ../src/ImageManager.py:805 ../src/ImageManager.py:809
msgid "Removing temp mounts..."
msgstr "Удаление времееных точек монтирования..."

#: ../src/SoftcamManager.py:162
msgid "Restart"
msgstr "Перезапуск"

#: ../src/MountManager.py:391
#, python-format
msgid "Restart %s %s."
msgstr ""

#: ../src/IPKInstaller.py:150
#, fuzzy
msgid "Restart GUI."
msgstr "Перезапуск"

#: ../src/BackupManager.py:155
msgid "Restore"
msgstr "Восстановление"

#: ../src/BackupManager.py:378
msgid "Restore Confirmation"
msgstr "Подтверждение восстановления"

#: ../src/plugin.py:181 ../src/RestoreWizard.py:132 ../src/RestoreWizard.py:137
#: ../src/RestoreWizard.py:144 ../src/RestoreWizard.py:150
#: ../src/RestoreWizard.py:156 ../src/RestoreWizard.py:160
#: ../src/RestoreWizard.py:186 ../src/RestoreWizard.py:190
#: ../src/RestoreWizard.py:199 ../src/RestoreWizard.py:255
#: ../src/RestoreWizard.py:259 ../src/RestoreWizard.py:263
#: ../src/RestoreWizard.py:267 ../src/RestoreWizard.py:271
#, fuzzy
msgid "Restore wizard"
msgstr "Мастер восстановления"

#: ../src/BackupManager.py:407 ../src/BackupManager.py:411
#: ../src/BackupManager.py:415
msgid "Restoring backup..."
msgstr "Восстановление резервных копий..."

#: ../src/BackupManager.py:443 ../src/BackupManager.py:447
msgid "Restoring plugins, this can take a long time..."
msgstr "Восстановление плагинов, это может занять много времени..."

#: ../src/BackupManager.py:435 ../src/BackupManager.py:439
msgid "Restoring plugins..."
msgstr "Восстановление плагинов..."

#: ../src/Multibootmgr.py:92
msgid "Retrieving image slots - Please wait..."
msgstr ""

#: ../src/ScriptRunner.py:65
msgid "Run"
msgstr "Выполнить"

#: ../src/ui.py:71
msgid "Run your shell scripts."
msgstr "Запуск скриптов"

#: ../src/MountManager.py:88
msgid "SDCARD: "
msgstr ""

#: ../src/SwapManager.py:271
#, fuzzy
msgid "SWAP file not found. You have to create the file before you try to activate it."
msgstr "Файл подкачки не найден. Вы должны создать файл, прежде чем активировать."

#: ../src/ui.py:72 ../src/SwapManager.py:102
#, fuzzy
msgid "SWAP manager"
msgstr "Файл подкачки"

#: ../src/SwapManager.py:121
#, fuzzy
msgid "SWAP place:"
msgstr "Расположение:"

#: ../src/SwapManager.py:123
#, fuzzy
msgid "SWAP size:"
msgstr "Размер: "

#: ../src/BackupManager.py:745 ../src/BackupManager.py:845
#: ../src/MountManager.py:352
msgid "Save"
msgstr "Сохранить"

#: ../src/ui.py:71 ../src/plugin.py:175 ../src/ScriptRunner.py:43
#, fuzzy
msgid "Script runner"
msgstr "Выполнить скрипт"

#: ../src/BackupManager.py:780
msgid "Select"
msgstr "Выбор"

#: ../src/BackupManager.py:245 ../src/BackupManager.py:248
#, fuzzy
msgid "Select a backup to restore:"
msgstr "Выберите резервную копию для восстановления:"

#: ../src/IPKInstaller.py:117
msgid "Select a package to install:"
msgstr "Выбор пакета для установки:"

#: ../src/ImageManager.py:1360
#, fuzzy, python-format
msgid "Select an image to download for %s:"
msgstr "Выбор имиджа для загрузки:"

#: ../src/ImageManager.py:257 ../src/ImageManager.py:262
#, fuzzy
msgid "Select an image to flash:"
msgstr "Выбор имиджа для загрузки:"

#: ../src/BackupManager.py:831
msgid "Select extra packages folder"
msgstr "Выберите папку дополнительных пакетов"

#: ../src/BackupManager.py:730
msgid "Select files/folders to backup"
msgstr "Выбор файлов/папок для создания резервной копии"

#: ../src/BackupManager.py:874
msgid "Select folder that contains plugins"
msgstr "Выберите папку содержащую плагины"

#: ../src/SwapManager.py:312
#, fuzzy
msgid "Select the SWAP file size:"
msgstr "Задайте размер файла подкачки:"

#: ../src/SoftcamManager.py:95
msgid "Select:"
msgstr "Выбор:"

#: ../src/ImageManager.py:769
#, fuzzy
msgid "Setting up..."
msgstr "Настройка..."

#: ../src/ui.py:45 ../src/SoftcamManager.py:80
msgid "Setup"
msgstr ""

#: ../src/MountManager.py:203
#, fuzzy
msgid "Setup mounts"
msgstr "Настройки монтирования"

#: ../src/MountManager.py:109 ../src/MountManager.py:115
#: ../src/MountManager.py:117 ../src/MountManager.py:119
#: ../src/MountManager.py:121
msgid "Size: "
msgstr "Размер: "

#: ../src/SoftcamManager.py:80
#, fuzzy
msgid "Softcam"
msgstr "Проверка эмулятора"

#: ../src/plugin.py:99 ../src/plugin.py:173 ../src/SoftcamManager.py:79
#, fuzzy
msgid "Softcam manager"
msgstr "Управление эмуляторами"

#: ../src/SoftcamManager.py:332
#, fuzzy
msgid "Softcam starting..."
msgstr "Запуск эмулятора..."

#: ../src/SoftcamManager.py:449
#, fuzzy
msgid "Softcam stopping..."
msgstr "Эмулятор останавливается..."

#: ../src/SoftcamManager.py:633
msgid "SoftcamCheck"
msgstr "Проверка эмулятора"

#: ../src/IPKInstaller.py:101
#, fuzzy
msgid "Sorry but that location does not exist or is not setup. Please set it up in the Backup manager setup menu."
msgstr "К сожалению указанный путь не существует или не настроен, пожалуйста, настройте в меню резервного копирования, Диспетчера резервирования"

#: ../src/BackupManager.py:563
#, fuzzy
msgid "Sorry the feeds are down for maintenance. Please try again later."
msgstr "Извините, ресурс закрыт на техническое обслуживание, пожалуйста, повторите попытку позже!"

#: ../src/RestoreWizard.py:149
#, fuzzy
msgid "Sorry the feeds are down for maintenance. Please try using Backup manager to restore plugins later."
msgstr "Извините, ресурс закрыт на техническое обслуживание. Пожалуйста, попробуйте использовать Менеджер резервного копирования для восстановления плагинов позже."

#: ../src/RestoreWizard.py:266
#, fuzzy
msgid "Sorry the feeds are down for maintenance. Please try using the Backup manager to restore plugins later."
msgstr "Извините, ресурс закрыт на техническое обслуживание. Пожалуйста, попробуйте использовать Менеджер резервного копирования для восстановления плагинов позже."

#: ../src/BackupManager.py:371 ../src/BackupManager.py:373
#: ../src/RestoreWizard.py:185 ../src/RestoreWizard.py:189
msgid "Sorry, but the file is not compatible with this image version."
msgstr ""

#: ../src/BackupManager.py:490
msgid "Sorry, but the restore failed."
msgstr "Извините, но восстановление не удалось."

#: ../src/SwapManager.py:306
#, fuzzy
msgid "Sorry, no physical devices that supports SWAP attached. Can't create SWAP file on network or fat32 file-systems."
msgstr "Не обнаружено устройств, поддерживающих создание файла подкачки. Невозможно создание на сетевых дисках или в файловой системе FAT32"

#: ../src/ImageManager.py:887
#, fuzzy
msgid "Sorry, not enough free RAM found, and no physical devices that supports SWAP attached. Can't create SWAP file on network or fat32 file-systems, unable to make backup."
msgstr "Не хватает свободной памяти, и нет устройств, поддерживающих создание файла подкачки. Невозможно создание на сетевых дисках или в файловой системе FAT32, невозможно создать резервную копию"

#: ../src/SoftcamManager.py:156
msgid "Start"
msgstr "Старт"

#: ../src/BackupManager.py:1149 ../src/BackupManager.py:1153
msgid "Starting..."
msgstr "Запуск..."

#: ../src/SwapManager.py:125
msgid "Status:"
msgstr "Состояние:"

#: ../src/SoftcamManager.py:158
msgid "Stop"
msgstr "Стоп"

#: ../src/MountManager.py:115
msgid "TB"
msgstr "ТВ"

#: ../src/IPKInstaller.py:107 ../src/IPKInstaller.py:115
#, fuzzy
msgid "Temp folder"
msgstr "Временный каталог"

#: ../src/BackupManager.py:1250
#, fuzzy
msgid "The backup location does not have enough free space."
msgstr "Выбранный путь не имеет достаточно свободного места"

#: ../src/ImageManager.py:850
#, fuzzy
msgid "The backup location does not have enough free space.\n"
msgstr "Выбранный путь не имеет достаточно свободного места"

#: ../src/MountManager.py:389
#, python-format
msgid ""
"The changes need a system restart to take effect.\n"
"Restart your %s %s now?"
msgstr ""

#: ../src/BackupManager.py:245 ../src/ImageManager.py:257
#, fuzzy
msgid "The chosen location does not exist, using /media/hdd."
msgstr "Выбранное место не существует, используйте /media/hdd"

#: ../src/BackupManager.py:268 ../src/ImageManager.py:271
#, fuzzy
msgid "There is a problem with this device. Please reformat it and try again."
msgstr "Возникла проблема с накопителем, пожалуйста, отформатируйте и попробуйте снова."

#: ../src/BackupManager.py:317
#, fuzzy
msgid "There is no backup to delete."
msgstr "Нет копии для удаления."

#: ../src/BackupManager.py:358
#, fuzzy
msgid "There is no backup to restore."
msgstr "Нет копии для восстановления."

#: ../src/ImageManager.py:325
#, fuzzy
msgid "There is no image to delete."
msgstr "Нет прошивки для удаления."

#: ../src/ImageManager.py:443
#, fuzzy
msgid "There is no image to flash."
msgstr "Нет прошивки для загрузки."

#: ../src/MountManager.py:304
#, fuzzy
msgid "This device is already mounted as HDD."
msgstr "Данное устройство уже смонтировано как HDD"

#: ../src/MountManager.py:130
msgid "Type: "
msgstr "Тип: "

#: ../src/MountManager.py:82
msgid "USB: "
msgstr "USB: "

#: ../src/MountManager.py:204
#, fuzzy
msgid "Un-mount"
msgstr "Отмонтир."

#: ../src/ImageManager.py:478
#, python-format
msgid ""
"Unzip error (also sent to any debug log):\n"
"%s"
msgstr ""

#: ../src/MountManager.py:384
#, fuzzy
msgid "Updating mount locations..."
msgstr "Обновление точек монтирования"

#: ../src/MountManager.py:227
msgid "Use as HDD"
msgstr "Использовать как HDD"

#: ../src/Multibootmgr.py:85
msgid "Use the cursor keys to select an installed image and then Erase button."
msgstr ""

#: ../src/BackupManager.py:200 ../src/BackupManager.py:341
#: ../src/BackupManager.py:384 ../src/ImageManager.py:186
#: ../src/ImageManager.py:359
#, fuzzy
msgid "View progress"
msgstr "Наблюдать выполнение"

#: ../src/ui.py:44 ../src/plugin.py:84 ../src/plugin.py:170
msgid "Vision"
msgstr "VISION"

#: ../src/plugin.py:183
#, fuzzy
msgid "Vision Backup manager"
msgstr "Vision Диспетчер резервирования"

#: ../src/plugin.py:184
#, fuzzy
msgid "Vision Image manager"
msgstr "Vision Диспетчер имиджей"

#: ../src/plugin.py:185
#, fuzzy
msgid "Vision Mount manager"
msgstr "Vision Диспетчер точек монтирования"

#: ../src/plugin.py:187
#, fuzzy
msgid "Vision SWAP manager"
msgstr "Vision Диспетчер файла подкачки"

#: ../src/plugin.py:186
#, fuzzy
msgid "Vision Script runner"
msgstr "Vision Запустить скрипт"

#: ../src/SwapManager.py:317
#, fuzzy
msgid "Wait please while creating SWAP file..."
msgstr "Пожалуйста, подождите, создаётся файла подкачки..."

#: ../src/SoftcamManager.py:188
msgid ""
"Wait please while scanning\n"
"for softcam's..."
msgstr ""
"Подождите, идёт поиск\n"
"эмуляторов..."

#: ../src/SwapManager.py:155
msgid "Wait please while scanning..."
msgstr "Подождите, идёт поиск..."

#: ../src/BackupManager.py:49 ../src/ImageManager.py:52
msgid "Weekly"
msgstr "Еженедельно"

#: ../src/RestoreWizard.py:103 ../src/RestoreWizard.py:106
msgid "YES, to restore plugins"
msgstr "Да, восстановить плагины"

#: ../src/RestoreWizard.py:97
msgid "YES, to restore settings"
msgstr "Да, восстановить настройки"

#: ../src/ImageManager.py:394
msgid ""
"You are about to flash an eMMC flash; we cannot take any responsibility for any errors or damage to your box during this process.\n"
"Proceed with CAUTION!:\n"
"Are you sure you want to flash this image:\n"
" "
msgstr ""

#: ../src/ImageManager.py:445
msgid "You have decided not to flash image."
msgstr ""

#: ../src/SwapManager.py:337
#, fuzzy
msgid "You have to create a SWAP file before trying to activate the autostart."
msgstr "Вы должны создать файл подкачки прежде, чем активировать автозапуск."

#: ../src/RestoreWizard.py:258
#, fuzzy, python-format
msgid "Your %s %s could not connect to the plugin feeds at this time. Please try using the Backup manager to restore plugins later."
msgstr "Извините, ресурс закрыт на техническое обслуживание. Пожалуйста, попробуйте использовать Менеджер резервного копирования для восстановления плагинов позже."

#: ../src/ImageManager.py:626
#, python-format
msgid ""
"Your %s %s is about to create a full image backup, this can take about 6 minutes to complete.\n"
"Do you want to allow this?"
msgstr ""

#: ../src/BackupManager.py:1086
#, python-format
msgid ""
"Your %s %s is about to run a backup of your settings and to detect your plugins.\n"
"Do you want to allow this?"
msgstr ""

#: ../src/BackupManager.py:554
#, python-format
msgid "Your %s %s is not connected to a network. Please check your network settings and try again."
msgstr ""

#: ../src/RestoreWizard.py:254
#, fuzzy, python-format
msgid "Your %s %s is not connected to a network. Please try using the Backup manager to restore plugins later when a network connection is available."
msgstr "Извините, ресурс закрыт на техническое обслуживание. Пожалуйста, попробуйте использовать Менеджер резервного копирования для восстановления плагинов позже."

#: ../src/BackupManager.py:572
#, python-format
msgid "Your %s %s is not connected to the Internet. Please check your network settings and try again."
msgstr ""

#: ../src/RestoreWizard.py:155
#, fuzzy, python-format
msgid "Your %s %s is not connected to the Internet. Please try using Backup manager to restore plugins later."
msgstr "Извините, ресурс закрыт на техническое обслуживание. Пожалуйста, попробуйте использовать Менеджер резервного копирования для восстановления плагинов позже."

#: ../src/RestoreWizard.py:262
#, fuzzy, python-format
msgid "Your %s %s is not connected to the Internet. Please try using the Backup manager to restore plugins later."
msgstr "Извините, ресурс закрыт на техническое обслуживание. Пожалуйста, попробуйте использовать Менеджер резервного копирования для восстановления плагинов позже."

#: ../src/ImageManager.py:427
#, python-format
msgid "slot%s - %s"
msgstr ""

#: ../src/Multibootmgr.py:128
#, python-format
msgid "slot%s - %s "
msgstr ""

#: ../src/ImageManager.py:427
#, python-format
msgid "slot%s - %s (current image)"
msgstr ""

#: ../src/MountManager.py:96 ../src/MountManager.py:109
#: ../src/MountManager.py:121
msgid "unavailable"
msgstr "недоступен"
